### ISSUE TITLE
very very big loss values

### ISSUE THREAD

**Issue by xiaogangLi** [hx173149/C3D-tensorflow#113]
Hi, I had a problem and need your help. I implemented C3D using Tensorflow.  When training it on my dataset, the loss values are very very big(e.g, 2.5e+19), which lead to a failure for training. I am sure the code by written correctly. I found the value Z (Z=WX+b) is gradually increasing，when add ing 3D convolution layers. Could you give some advice? Thank you!
